src/components/Match/matchColumns.jsx

```diff
@@ -1236,10 +1236,8 @@ export default (strings) => {
         totalDuration.push(duration);
       }
     });
-    let total = 0;
-    for (let i = 0; i < totalDuration.length; i += 1) {
-      total += totalDuration[i];
-    }
+
+    const total = totalDuration.reduce((a, b) => a + b, 0);
```

Comment:
I'm fine with this, but the original might be a little easier to read. Not everyone is familiar with the .reduce function.

Reply:
True. This is as basic as a reduce function gets though, so even if someone is not familiar with it, he can probably still understand it (also probably a good way to learn it?)